_sass/general.scss

```diff
@@ -16,6 +16,7 @@ body {
     font-size: $size-font-normal;
     line-height: 1.75em;
     color: $color-text-normal;
+		background-color: $color-white;
```

Comment:
Why did you add `background-color`? I don't see any usage of it.

Reply:
When users set default browser colors to e.g. white-on-black and the website sets a dark custom foreground color, the background also needs adjustment. Otherwise you'll have dark text on a dark background.

Reply:
Wrote a bit more about this issue in the "Default colors" section of https://seirdy.one/posts/2020/11/23/website-best-practices/#default-colors